src/onnxruntime.cc

```diff
@@ -2484,6 +2521,22 @@ TRITONBACKEND_Initialize(TRITONBACKEND_Backend* backend)
   // Onetime initialization for the onnxruntime loader.
   RETURN_IF_ERROR(OnnxLoader::Init(backend_config));
 
+  std::unique_ptr<BackendConfiguration> lconfig(new BackendConfiguration());
```

Comment:
We are already reading the backend_config in `OnnxLoader::Init` for the thread pool config. Does it make sense to parse the backend_config once and pass around the required information to these functions?

Reply:
I thought about that and came to the conclusion it might be easier to maintain if all backends had the same look to them.  I'll go ahead and change this.

Reply:
After the attempted change, I found that the code layout would look worse than it does right now. To make the suggested change, the `BackendConfiguration` struct would have to move into onnxruntime_loader.h or I would have to create a new header file onnxruntime.h which only holds the definition of the struct.  I'm not a fan of both of these options since it diverges the general backend code base without a good reason for it, imo. I can make an argument for processing onnx configs in the Init() function and parsing general backned parameters in onnxruntime.cc but let me know.

Reply:
We can pass only the required parameters from backend config to OnnxLoader::Init().

Reply:
Resolving the comment for now to limit the scope of the change.